src/VstsSyncMigrator.Core/Execution/MigrationContext/TestVeriablesMigrationContext.cs

```diff
@@ -47,38 +46,38 @@ protected override void InternalExecute()
             TestManagementContext SourceTmc = new TestManagementContext(Engine.Source);
             TestManagementContext targetTmc = new TestManagementContext(Engine.Target);
             List<ITestVariable> sourceVars = SourceTmc.Project.TestVariables.Query().ToList();
-            Trace.WriteLine(string.Format("Plan to copy {0} Veriables?", sourceVars.Count));
+            Log.LogInformation("Plan to copy {0} Veriables?", sourceVars.Count);
```

Comment:
Do we not want to name this like the other places?

Reply:
There is a couple places where we've just got {0} from the previous string format usage

Reply:
Yes, should have names but my first pass was "move to log" not "make log awesome" :)